### PR TITLE
i18n(fr): update `guides/imports.mdx` & `migrate-to` guides

### PR DESCRIPTION
**Pull request by ArmandPhilippot**
#### Description (required)

Adds changes from #9240 to the French translations of:
* the `migrate-to` guides (including fixes from Junseong and #10332 )
* `guides/imports.mdx` (including #10140, #10310 and #10315)

> [!NOTE]
> I haven't updated some links in the `migrate-to` guides to avoid broken links:
> * those targeting `api-reference.mdx` 
> * those targeting `install-and-setup.mdx`
>
> I think it's better to do it while updating these files, but I can update them here if needed...

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/migrate-to-astro/from-create-react-app.mdx [hunk 1]
```diff
@@ -346,7 +348,7 @@ const randomUser = data.results[0];
 ---
 ```
 
-En savoir plus sur [l'importation de fichiers locaux avec `Astro.glob()`](/fr/guides/imports/#astroglob), [l'interrogation à l'aide de l'API Collections](/fr/guides/content-collections/#interroger-les-collections) ou [la récupération de données distantes](/fr/guides/data-fetching/).
+En savoir plus sur l'importation de fichiers locaux avec [`import.meta.glob()`](/en/guides/imports/#importmetaglob), [l'interrogation à l'aide de l'API Collections](/fr/guides/content-collections/#interroger-les-collections) ou [la récupération de données distantes](/fr/guides/data-fetching/).
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/migrate-to-astro/from-create-react-app.mdx:351]: Wrong locale...
```suggestion
En savoir plus sur l'importation de fichiers locaux avec [`import.meta.glob()`](/fr/guides/imports/#importmetaglob), [l'interrogation à l'aide de l'API Collections](/fr/guides/content-collections/#interroger-les-collections) ou [la récupération de données distantes](/fr/guides/data-fetching/).
```

**Review by thomasbnt — CHANGES_REQUESTED** [review]
Bold text is missing

**Comment by ArmandPhilippot** [comment]
@thomasbnt I also added the changes from #10332 which is not yet merged! This will avoid having to re-edit these files... So check if the other one is merged before this one. 😅

**Comment by thomasbnt** [comment]
> @thomasbnt I also added the changes from #10332 which is not yet merged! This will avoid having to re-edit these files... So check if the other one is merged before this one. 😅
> 
> Edit: all good, this is merged! 😄

Oh good!

**Review by thomasbnt — APPROVED** [review]
LGTM! Thanks @ArmandPhilippot for your contribution!

**Pull request merged**